client/service/test/src/lib.rs

```diff
@@ -312,11 +310,7 @@ pub fn connectivity<G, E, Fb, F, Lb, L>(
 	const NUM_LIGHT_NODES: usize = 5;
 
 	let expected_full_connections = NUM_FULL_NODES - 1 + NUM_LIGHT_NODES;
-	let expected_light_connections = if light_node_interconnectivity {
-		expected_full_connections
-	} else {
-		NUM_FULL_NODES
-	};
+	let expected_light_connections = NUM_FULL_NODES;
```

Comment:
Should this be `NUM_LIGHT_NODES`?

Reply:
It is a number of connections at light nodes. Light nodes only connect to full nodes => `NUM_FULL_NODES`.